### ISSUE TITLE
AWS Lambda Test Tool - error parsing JSON file appsettings.Development.json

### ISSUE THREAD

**Issue by nathanvj** [aws/aws-lambda-dotnet#1371]
### Describe the bug

I have an AWS project with two Lambda Functions.

The project has a reference to my main project, which contains appsettings.

When I start the AWS Lambda Test Tool, it gives me the following error in my console:
```
Error parsing JSON file: ...MyProject.Lambda\bin\Debug\net6.0\appsettings.Development.json
```

And in my browser:
```
FileNotFoundException: Could not load file or assembly 'MyProject.Lambda, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
```

When I remove the reference to my main project, the error goes away. 

**Please note that when I deploy the AWS Lambda, and test invoke it from AWS Console it works fine.**

### Expected Behavior

I want to be able to load `appsettings` when using the Lambda Test Tool.

### Current Behavior

Console:
```
Error parsing JSON file: ...MyProject.Lambda\bin\Debug\net6.0\appsettings.Development.json
```

Browser:
```
FileNotFoundException: Could not load file or assembly 'MyProject.Lambda, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
```

### Reproduction Steps

Project A (NET6):
```
Business logic, services, installers, etc
appsettings.json
appsettings.Development.json
appsettings.Production.json
```

Project B (Lambda .NET 6):
```
References Project A.
Two AWS Lambda functions.
```

### AWS .NET SDK and/or Package version used

Amazon.Lambda.CloudWatchEvents 4.0.0
Amazon.Lambda.Core 2.1.0
Amazon.Lambda.Serializiation.SystemTextJson 2.3.0

### Targeted .NET Platform

.NET 6

### Operating System and version

Windows 10

**Comment by nathanvj** [comment]
Anyone who can help me out?

**Comment by ashishdhingra** [comment]
@nathanvj Good evening. Could you share sample code solution to reproduce the issue?

**Comment by shrinaik** [comment]
One of the reasons for this issue could be case. Can you once verify if the Case of the appsettings.Development.json file name matches with wherever it is referred be it in code or anyother config file. This was one of the reason I got the same error.

**Comment by nathanvj** [comment]
Sorry I was quite busy and only just got the time to create an example repository.

I originally thought the reason the error occured was due to the `appsettings` that I was loading (the error in my terminal said it could not be parsed), and I remember it working fine before I added it.

Now after creating this minimal reproducible example, I found out that I get this error regardless (even if I comment part out):
```
FileNotFoundException: Could not load file or assembly 'LambdaTestToolError.Lambda, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
```

I would be very grateful if someone could take a look. The example repository can be found here:
[https://github.com/nathanvj/LambdaTestToolError](https://github.com/nathanvj/LambdaTestToolError)

Kind regards
